Makefile

```diff
@@ -4,6 +4,8 @@ src_dir=~/cafebabel.com
 venv_dir=~/cafebabel.com/venv
 branch=master
 LOGS=access
+prod=prod@91.194.60.65
```

Comment:
I'd suggest to call this and the line below something like `old_prod`, or it's comfusing with the future _prod_ env.